.travis.yml

```diff
@@ -37,7 +37,7 @@ rvm:
   - 1.8.7
   - 2.4.2
   - 2.3.5
-  - 2.2
+  - 2.2.7
```

Comment:
`2.2` automatically tracks the latest, but yes I forgot to update the matrix when I did everything else, can you back out the 2.2 changes and then this can be merged?

Reply:
Sure. Just reverted around 2.2 changes as https://github.com/rspec/rspec-rails/pull/1906/commits/c6a47a0ede8184478da67051ae481b9c2abf9183